requirements.txt

```diff
@@ -0,0 +1,3 @@
+bleach==2.1.4
```

Comment:
Can you move this file to the `requirements/` folder?

Reply:
Sure, done.